apps/src/maze/mazeMap.js

```diff
@@ -117,25 +120,10 @@ MazeMap.getAllStaticGrids = function (variableGrid) {
   return grids;
 };
 
-/**
- * @return {boolean}
- */
-MazeMap.prototype.hasMultiplePossibleGrids = function () {
-  return this.staticGrids.length > 1;
-};
 
+MazeMap.deserialize = (serializedValues, cellClass) => new MazeMap(serializedValues.map(row => row.map(cellClass.deserialize)));
```

Comment:
Nit: maybe worth making this mutli-line to fit in the 80 char soft limit.  (One more below.)